ckanext/multilang/plugin.py

```diff
@@ -287,3 +291,94 @@ def after_create(self, context, resource):
                 ResourceMultilang.persist(resource, lang)
 
 
+LOCALIZED_RESOURCES = 'ckanext.multilang.localized_resources'
+
+class MultilangResourcesPlugin(plugins.SingletonPlugin, toolkit.DefaultDatasetForm):
+
+    ML_ENABLED = 'multilang' in (config.get('ckan.plugins') or '').split(' ')
+    LOCALIZED_RESOURCES_ENABLED = toolkit.asbool(config.get(LOCALIZED_RESOURCES))
+    LOCALIZED_RESOURCES_IN_DCATAPIT = 'dcatapit_pkg' in (config.get('ckan.plugins') or '').split(' ')
```

Comment:
There's a reference to a less general plugin (dcatapit) in a more general one (multilang).
This is a Bad Thing: What if we need multilang in another plugin which conflict? Shall we add exceptions here as well?
Please consider using a configuration line which lists "blacklisted" plugins

Reply:
True, this is dirty hack on IDatasetForm limitation. The purpose was to have localized resource urls working for multilang standalone and multilang+dcatapit combined deployments. I didn't know any other use cases, hence dcatapit referenced directly.

I'd opt for white-list approach instead, and list extensions supported. If extension is in WL, we assume it will take over IDatasetForm implementation, otherwise ML extension is responsible for this.